src/modules/blockchainApplication/manage/store/reducer.test.js

```diff
@@ -31,7 +33,7 @@ describe('BlockchainApplication reducer', () => {
         name: 'New app',
         chainID: 'aq02qkbb35u4jdq2szo6ytre',
```

Comment:
this will be integer, just for consistency can you update all the mock data to integer? sample value `0001`